docs/dg/dev/backend-development/audit-logs/audit-logs.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Audit logs
```

Comment:
@mikost7 We discussed with you that the current state of the feature is security oriented.
Could we please make sure that this is clearly and early visible from this document?
Right now this is mentioned in the `Default plugins` section only.